### PR TITLE
feat: move editable select into its own component

### PR DESCRIPTION
**Pull request by shogunpurple**
Moving the editable select out into it's own datalist component

### CODE REVIEW

File: src/Form/DataList.svelte [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  import Icon from "../Icons/Icon.svelte";
+  import { createEventDispatcher } from "svelte";
+
+  export let value = "";
+  export let name = undefined;
+  export let thin = false;
+  export let secondary = false;
+  export let outline = false;
+  export let disabled = false;
+  export let editable = false;
```
Comment (mjashanks) [inline src/Form/DataList.svelte:12]: afaik, DataList should always be editable... thats what sets it apart from Select. We should probably remove this prop

**Review by mjashanks — CHANGES_REQUESTED** [review]
DataList should always be editable

File: src/Form/DataList.svelte (hunk 1 above)
Comment (shogunpurple) [reply src/Form/DataList.svelte:12]: Yeah, makes sense was just copy pasta from `Select`

**Review by mjashanks — APPROVED** [review]
LGTM!

**Pull request merged**